vue-components/src/components/OptionsMenu.vue

```diff
@@ -91,13 +105,26 @@ export default Vue.extend( {
 					this.$emit( 'esc' );
 					break;
 				case 'ArrowUp':
-					this.keyboardHoveredItemIndex = Math.max( 0, this.keyboardHoveredItemIndex - 1 );
+					if ( this.allowLooping && this.keyboardHoveredItemIndex == 0 ) {
+						// loop to the bottom of the menu
+						this.keyboardHoveredItemIndex = this.menuItems.length - 1;
+					} else {
+						this.keyboardHoveredItemIndex = Math.max( 0, this.keyboardHoveredItemIndex - 1 );
+					}
+
+					this.keyboardScroll();
 					break;
 				case 'ArrowDown':
-					this.keyboardHoveredItemIndex = Math.min(
+					if ( this.allowLooping && this.keyboardHoveredItemIndex == this.menuItems.length - 1 ) {
+						// go back to the top of the menu
+						this.keyboardHoveredItemIndex = 0;
+					} else {
+						this.keyboardHoveredItemIndex = Math.min(
 						this.menuItems.length - 1,
```

Comment:
I would have expected the linter to complain about the missing indentation :thinking: 

cc @itamargiv

Reply:
:+1: I don't think there's a particular rule currently set for that. I would suggest opening a separate task and PR to implement this rule (the "CallExpression" option on the ["indent"](https://eslint.org/docs/rules/indent) rule) and in the meanwhile just suggest the preferred indentation.